demo/demo-tekton.sh

```diff
@@ -221,17 +239,33 @@ printf "Templates: "
 oc delete --all template.template.openshift.io -n "${app_namespace}" 2> /dev/null || echo "...done"
```

Comment:
first step (clean up prior to execution) does not delete everything `oc get all --namespace basic-python-tekton`
```
        NAME                          TYPE        CLUSTER-IP      EXTERNAL-IP   PORT(S)    AGE
        service/basic-python-tekton   ClusterIP   172.30.55.112   <none>        8080/TCP   9m33s

        NAME                                           HOST/PORT                                                                                        PATH   SERVICES              PORT    TERMINATION   WILDCARD
        route.route.openshift.io/basic-python-tekton   basic-python-tekton-basic-python-tekton.apps.cluster-wdh11092022b.wdh11092022b.mg.dog8code.com          basic-python-tekton   <all>                 None
```
does this matter?

Reply:
It's demo so to show the application is up and running purpose only.